src/test/java/org/opensearch/knn/index/mapper/KNNVectorFieldMapperTests.java

```diff
@@ -239,6 +257,29 @@ public void testBuilder_parse_fromKnnMethodContext_luceneEngine() throws IOExcep
             () -> builderInvalidDimension.build(new Mapper.BuilderContext(settings, new ContentPath()))
         );
         assertEquals("Dimension value cannot be greater than [1024] but got [2000] for vector [test-field-name]", ex.getMessage());
+
+        XContentBuilder xContentBuilderUnsupportedParam = XContentFactory.jsonBuilder()
+            .startObject()
+            .field("type", "knn_vector")
+            .field("dimension", dimension)
+            .startObject(KNN_METHOD)
+            .field(NAME, METHOD_HNSW)
+            .field(METHOD_PARAMETER_SPACE_TYPE, SpaceType.L1)
+            .field(KNN_ENGINE, LUCENE_NAME)
+            .startObject(PARAMETERS)
+            .field("RANDOM_PARAM", 0)
+            .endObject()
+            .endObject()
+            .endObject();
+
+        expectThrows(
```

Comment:
What are we testing for as invalid here? I think I see a couple things.

Reply:
Right, it should be only the random param name, let me update the test case class